cfpb/cfgov-refresh
# Add HMDA Historic Data page

Adds a page that will replace our current [HMDA data page](https://www.consumerfinance.gov/data-research/hmda/explore). It includes a form that controls a table linking to CSV files hosted on S3.

## Additions

- `HmdaHistoricDataPage` page type
- `HmdaFilterableForm` form with location, record type and description fields.
- `loan_file_metadata.py` file that provides the record count and file size of every CSV.
- Templates 'n stuff

## Testing

1. Create a `Hmda historic data page` page
1. Optionally add any content to the page
1. Publish the page and you should see something like the screenshot below.

## Todos

- The helper text under the form fields isn't included. Implementing that is proving to be tricky.

## Screenshots

![b50ed780-4b26-11e9-95bb-6580e7c7ed67](https://user-images.githubusercontent.com/1060248/55099777-b98f3f00-5096-11e9-87f6-34ae71745b5f.jpg)
